plugins/gstperf.c

```diff
@@ -469,6 +469,8 @@ gst_perf_transform_ip (GstBaseTransform * trans, GstBuffer * buf)
         (GstElement *) perf,
         gst_message_new_info ((GstObject *) perf, perf->error,
             (const gchar *) info));
+
+    GST_INFO_OBJECT (perf, "%s", info);
```

Comment:
It could be moved as `GST_LOG_OBJECT` since it is called on `transform_ip` function, which is called for each buffer.


https://developer.ridgerun.com/wiki/index.php/GStreamer_Debugging#:~:text=The%20GStreamer%20debugging%20levels%20include%3A,-0&text=No%20debug%20information%20is%20output.&text=Logs%20all%20fatal%20errors.,conditions%20that%20triggered%20the%20error.

Reply:
This only occurs once every second (when the perf prints the info). Check the conditional in line 421.